app/config/routes.yaml

```diff
@@ -0,0 +1,12 @@
+index:
+   path: /
+   defaults: { _controller: 'App\Controller\HelloController::start' }
+
+# app_controller:
```

Comment:
If this is commented, can't it be deleted?

Reply:
Yes, but it was generated by symfony